core/platforms/packaging/bareos.spec

```diff
@@ -143,6 +143,7 @@ BuildRequires: systemd
 # see https://en.opensuse.org/openSUSE:Systemd_packaging_guidelines
 %if 0%{?suse_version} >= 1210
 BuildRequires: systemd-rpm-macros
+BuildRequires: libudev-devel
```

Comment:
What's the point in adding this?
I'm asking, because we build successfully without it...

Reply:
Without that line I get
```
home:bmwiedemann:bareos/bareos # osc build --noservice --release=2 15.3 bareos.spec
Building bareos.spec for 15.3/x86_64
Getting buildconfig from server and store to /space/osc/h/home:bmwiedemann:bareos/bareos/.osc/_buildconfig-15.3-x86_64
Getting buildinfo from server and store to /space/osc/h/home:bmwiedemann:bareos/bareos/.osc/_buildinfo-15.3-x86_64.xml
buildinfo is broken... it says:
unresolvable: conflict for providers of systemd-mini = 246.16-150300.7.39.1 needed by udev-mini
      (provider systemd-mini conflicts with systemd)
```

That might be some artifact of those udev-mini and systemd-mini packages in OBS that are not published.

We can omit it for now, if you like.

Reply:
Yeah, that explains why you'd need it.
TBH, I would like to keep workarounds for specific build-systems out of the SPEC as much as possible. If you have a way of working around the issue you're describing I'd rather not add it. History has shown this kind of workaround usually stay there forever.

The rest of the PR looks like a great improvement. So if it passes CI, I'm happy to merge it.

Reply:
I dropped this patch from the PR.